libs/cri-api-service/src/main/java/uk/gov/di/ipv/core/library/criapiservice/CriApiService.java

```diff
@@ -68,6 +78,7 @@ public class CriApiService {
     private final SignerFactory signerFactory;
     private final SecureTokenHelper secureTokenHelper;
     private final Clock clock;
+    private final HttpClient httpClient;
```

Comment:
There'll be a conflict here with https://github.com/govuk-one-login/ipv-core-back/pull/2672, but should be easy enough to resolve - we can share the HTTP client between the two.

Reply:
I've moved the method which uses this into its own service now so hopefully shouldn't be any conflicts here

Reply:
oh I just had a proper look at the PR, is it for all requests to the CRIs? I'll update this if the other one gets merged first

Reply:
I think this is probably irrelevant now you've pulled it into a separate service - I don't mind that, the JWKS endpoint isn't so much of a 'CRI API' as the token/credential endpoints.